lib/processor.js

```diff
@@ -5,6 +5,9 @@ const uglify = require('uglify-es');
 const getConfig = require('./config');
 const config = getConfig();
 
+const MAX_LINE_COUNT_OF_SHORT_EXAMPLES = 10;
+const MIN_LINE_COUNT_OF_TALL_EXAMPLES = 14;
```

Comment:
I would call these:
```
MAX_LINE_COUNT_OF_SHORT_JS_EXAMPLES
MIN_LINE_COUNT_OF_TALL_JS_EXAMPLES
```

...or something, because this is specific to JS.